config/initializers/sentry.rb

```diff
@@ -1,5 +1,11 @@
 # Configure sentry
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = ENV['SENTRY_DSN']
-  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+  config.send_default_pii = true
```

Comment:
We're making a judgment call here to filter stuff via activesupport rather than rely on the out-of-codebase settings. The PII here refers generally to stuff like user IP and sending a payload at all, which we do sorta want I think.